### ISSUE TITLE
#384983798------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1561]
<strong>Source:</strong>`<a id="edit-button-921a0a859b514f3684d6e1e917363e67" href="/layers/921a0a859b514f3684d6e1e917363e67/edit" aria-describedby="calcite-tooltip-aebf32e3-a791-fd0e-cbf7-a56cfba3a342">`

<strong>Element Location: </strong>#edit-button-921a0a859b514f3684d6e1e917363e67

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,wcag244,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/link-name?application=axeAPI
